app/views/dashboard/_distribution_widget.html.erb

```diff
@@ -1,4 +1,11 @@
+<% origin = @data.settings[:origin]
+   interval_setting =  if origin == 'All'
+                         Setting[:outofsync_interval]
+                       else
+                         Setting[:"#{origin.downcase}_interval"]
```

Comment:
if plugin adds origin but does not define the setting, we get 500 for this widget, I guess we should fallback to `Setting[:outofsync_interval]`?

Reply:
also there's another problem, my existing widgets don't have origin set so it fails with `nil.downcase` and since the widget fails to load, I can't even close it

so perhaps `elsif origin.present?` and default to Setting[:outofsync_interval] in else